woocommerce/wc-admin
# Build: Add `cross-env` to allow building on Windows

This branch adds [the `cross-env` package](https://www.npmjs.com/package/cross-env) to the repo so that we can safely set the `NODE_ENV` to production for the build script. I ran into this issue setting up a Windows VM + dev site, so others will probably have trouble too.

**To test**

- Check out branch
- Make sure `npm run build` works
- If you have a windows dev machine, try running this in command prompt, it should still complete
